tests/std/tests/P0645R10_text_formatting_death/test.cpp

```diff
@@ -17,8 +17,8 @@ void test_case_advance_no_range() {
     context.advance_to(other_format_string.begin());
 }
 
-void test_case_zero_dynamic_width() {
-    (void) format("{:{}}", 42, 0);
+void test_case_negative_dynamic_width() {

```

Comment:
I've brought this back as a failure case for negative dynamic width. (No change requested.)